core-feature-pack/common/src/main/resources/content/bin/standalone.sh

```diff
@@ -332,6 +332,10 @@ echo ""
 echo "========================================================================="
 echo ""
 
+if [[ ! $SERVER_OPTS =~ -Djboss.server.base.dir && ! $JAVA_OPTS =~ -Djboss.server.base.dir ]]; then
```

Comment:
Notice the shebang is `sh` not `bash`, so we cannot use some of the enhanced bash features. Even if your current shell allows it, it is not a POXIS standard, for example, it will fail in dash.

Reply:
Oh, I see, I will rework it accordingly, Thanks @yersan